src/node.cc

```diff
@@ -168,6 +169,7 @@ static uv_async_t dispatch_debug_messages_async;
 static node::atomic<Isolate*> node_isolate;
 static v8::Platform* default_platform;
 
+int v8_thread_pool_size = v8_default_thread_pool_size;
```

Comment:
Would it work to ditch `v8_default_thread_pool_size`, and just initialize this to `DefaultPlatform::kMaxThreadPoolSize`, like @ofrobots suggested?

I think you'll want to group this variable with the variables declared starting at line 134.